src/language/lang_hu.php

```diff
@@ -0,0 +1,333 @@
+	'Soon' => 'Hamarosan',
+	'Upcoming Game' => 'Következő játék',
+	'_days' => '_nap',
+	'_hours' => '_óra',
+	'_minutes' => '_perc',
+	'_seconds' => '_másodperc',
+	'Official CTF Rules' => 'A hivatalos CTF szabályok',
+	'Following actions are prohibited, unless explicitly told otherwise by event Admins.' => 'A következő tevékenységek tilosak, kivéve ha a szervezők külön engedélyt adnak rá.',
+	'Rule' => 'Szabály',
+	'Cooperation' => 'Csapatok közötti együttműködés',
+	'No cooperation between teams with independent accounts. Sharing of keys or providing revealing hints to other teams is cheating, don’t do it.' => 'A csapatok közötti együttműködés tilos. Megoldások megosztása vagy más csapatok segítése csalás, ne csináljátok.',
+	'Attacking Scoreboard' => 'Megtámadni az eredményjelzőt',
+	'No attacking the competition infrastructure. If bugs or vulns are found, please alert the competition organizers immediately.' => 'Tilos megtámadni a verseny infrastruktúráját. Ha bug-ot vagy sebezhetőséget találtok, kérjük azonnal értesítsétek a szervezőket.',
+	'Sabotage' => 'Más csapatok akadályozása',
+	'Absolutely no sabotaging of other competing teams, or in any way hindering their independent progress.' => 'Tilos más csapatokat akadályozni vagy hátráltatni.',
+	'Bruteforcing' => 'Bruteforcing',
+	'No brute forcing of challenge flag/ keys against the scoring site.' => 'Tilos a megoldás beküldő rendszert automatizált nagyszámú próbálgatással (bruteforce-szal) terhelni.',
+	'Denial Of Service' => 'Denial Of Service',
+	'DoSing the CTF platform or any of the challenges is forbidden.' => 'Tilos túlterheléses támadást (DOS) indítani a CTF rendszer vagy bármelyik feladat ellen.',
+	'Legal' => 'Korlátozott felelősségi nyilatkozat',
+	'Disclaimer' => 'Legal Disclaimer',
+	'By participating in the contest, you agree to release Facebook and its employees, and the hosting organization from any and all liability, claims or actions of any kind whatsoever for injuries, damages or losses to persons and property which may be sustained in connection with the contest. You acknowledge and agree that Facebook et al is not responsible for technical, hardware or software failures, or other errors or problems which may occur in connection with the contest.' => 'A versenyen való részvétellel a résztvevők elfogadják, hogy a Facebook, a Facebook munkatársai valamint a verseny szervezői mentesek minden felelősségtől abban az esetben, ha sérülés, kár vagy veszteség éri a résztvevőket a verseny során. By participating in the contest, you agree to release Facebook and its employees, and the hosting organization from any and all liability, claims or actions of any kind whatsoever for injuries, damages or losses to persons and property which may be sustained in connection with the contest. You acknowledge and agree that Facebook et al is not responsible for technical, hardware or software failures, or other errors or problems which may occur in connection with the contest.',
```

Comment:
Here's my take on it: "A versenyen való részvétellel elfogadod, hogy a Facebook, a Facebook munkatársai valamint a verseny szervezői mentesek minden felelősségtől abban az esetben, ha sérülés, kár vagy veszteség ér bárkit vagy bármit a verseny során vagy a versennyel kapcsolatban. Elfogadod, és egyetértesz azzal, hogy a Facebook és társai nem felelősek semmilyen technikai, hardver- vagy szoftverhibákért vagy bármilyen más problémáért vagy hibáért ami a versennyel kapcsolatban felmerül."
Wdyt?